jobs/mattermost/templates/config/initial-config.json.erb

```diff
@@ -190,7 +201,8 @@
         "UseChannelInEmailNotifications": <%= p("mattermost.EmailSettings.UseChannelInEmailNotifications") %>,
         "LoginButtonColor": "<%= p("mattermost.EmailSettings.LoginButtonColor") %>",
         "LoginButtonBorderColor": "<%= p("mattermost.EmailSettings.LoginButtonBorderColor") %>",
-        "LoginButtonTextColor": "<%= p("mattermost.EmailSettings.LoginButtonTextColor") %>"
+        "LoginButtonTextColor": "<%= p("mattermost.EmailSettings.LoginButtonTextColor") %>",
```

Comment:
seems we have this twice.   does that seem right to you?  see line 280

Reply:
There should be a `LoginButtonTextColor` attribute for both EmailSettings and LdapSettings - is that what you're talking about Tejay?

Reply:
It sure is.  I was hoping it was something like that.  Sounds like we're fine then.